nexus/reconfigurator/planning/src/blueprint_builder/builder.rs

```diff
@@ -183,6 +231,10 @@ impl fmt::Display for Operation {
     }
 }
 
+fn zone_name(zone: &BlueprintZoneConfig) -> String {
+    format!("oxz_{}_{}", zone.zone_type.kind().zone_prefix(), zone.id)
```

Comment:
Do we not have this as a helper function somewhere commonly accessible already? Or is the fact that we need to reconstruct here a sign that something is leaking out?

Reply:
I think this is also defined here: https://github.com/oxidecomputer/omicron/blob/356c9bf624aba3faf0b9b35239bfef4704f743c3/illumos-utils/src/running_zone.rs#L940-L956

But yeah, this is decidedly a leaky abstraction. I'll add a dependency on `illumos-utils` and use a shared helper.

Updated in 24b0341b357c7e7ec98ae5026b57a629bf6dc5cc